.env

```
@@ -0,0 +1,3 @@
+DATABASE_URL=postgresql://postgres:postgres@db:5432/apae_comemorativo
+BLOB_READ_WRITE_TOKEN=supersecrettoken
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BLOB_READ_WRITE_TOKEN key should go before the DATABASE_URL key

(UnorderedKey)